KIF Tests/TableViewTests_ViewTestActor.m

```diff
@@ -148,7 +149,7 @@ - (void)testButtonAbsentAfterSetHidden
 
 - (void)testEnteringTextIntoATextFieldInATableCell
 {
-    [[viewTester usingLabel:@"TextField"] enterText:@"Test-Driven Development"];
+    [[viewTester usingLabel:@"TextField"] enterText:@"Testing"];
```

Comment:
It might be a good idea to add a const for the `@"Testing"` string literal, and reference it from both places - with a comment explaining that it should not be changed to not mess with the auto-correct?